2.arrayOccurenceFilter.js

```diff
@@ -11,7 +11,8 @@
 // occurrenceFilter([1, 3, 4, 1, 9, 1, 3, 4, 3, 1, 2], 2) returns [1, 3, 4] 
 
 const occurrenceFilter = (arr, n) => {
-
+  let count = 0;
+  for(let i = 0, i < arr)
 }
```

Comment:
Please revisit all count occurance problems. This pattern of making a counter object is extremely common and needs to become second nature.